packages/backend/src/workers/action.ts

```diff
@@ -126,7 +126,7 @@ worker.on('failed', async (job, err) => {
       .throwIfNotFound()
     const errorDetails = await sendErrorEmail(flow)
     logger.info(`Sent error email for FLOW ID: ${job.data.flowId}`, {
-      errorDetails,
+      errorDetails: { ...errorDetails, ...job.data },
```

Comment:
Don't mirror the variable name `errorDetails` to minimise confusion.

```typescript
    const emailErrorDetails = await sendErrorEmail(flow)
    logger.info(`Sent error email for FLOW ID: ${job.data.flowId}`, {
      errorDetails: { ...emailErrorDetails, ...job.data },
```

Reply:
ok got it, thanks!